crossplane-provider-aws.yaml

```diff
@@ -77,22 +50,50 @@ data:
       sqs: provider-aws-sqs
 
 subpackages:
-  - name: crossplane-provider-aws-${{range.key}}
+  - name: ${{package.name}}-${{range.key}}
     range: packages
     pipeline:
+      - uses: go/build
+        with:
+          packages: ./cmd/provider/${{range.key}}
+          output: ${{range.value}}
+          ldflags: |
+            -X github.com/crossplane-contrib/provider-upjet-aws/internal/version.Version=v${{package.version}}
       - runs: |
-          up xpkg xp-extract xpkg.upbound.io/upbound/${{range.value}}:v${{package.version}}
           mkdir -p "${{targets.subpkgdir}}"
-          gunzip out.gz -c > "${{targets.subpkgdir}}"/package.yaml
+          crane export xpkg.upbound.io/upbound/${{range.value}}:v1.23.1 - | tar -xOf - package.yaml > ${{targets.subpkgdir}}/package.yaml
```

Comment:
If `latest` isn't an option, we should provide a variable towards the top of the package manifest